pages/common/act.md

```diff
@@ -19,7 +19,7 @@
 
 `act {{[-j|--job]}} {{job_id}}`
 
-- Do [n]ot actually run the actions (i.e. a dry run):
+- Do not actually run the actions (i.e. a dry run):
```

Comment:
This is appropriate because the short option differs greatly from the long option. Consulting the internet, `-n` seems to be specifically short for "no-op", so maybe the description could be improved.